src/storage/recovery/recovery_manager.cpp

```diff
@@ -58,11 +71,20 @@ void RecoveryManager::RecoverFromLogs() {
       }
 
       case (LogRecordType::COMMIT): {
+        //STORAGE_LOG_INFO("[Recovery Manager]: Processing COMMIT logs");
         NOISEPAGE_ASSERT(pair.second.empty(), "Commit records should not have any varlen pointers");
         auto *commit_record = log_record->GetUnderlyingRecordBodyAs<CommitRecord>();
 
         // We defer all transactions initially
-        deferred_txns_.insert(log_record->TxnBegin());
```

Comment:
Is there a reason for this?

Reply:
I didn't remove it, I moved it into the conditionals below.